core/io/file_access_compressed.cpp

```diff
@@ -130,34 +130,37 @@ void FileAccessCompressed::_close() {
 			f->store_32(0); //compressed sizes, will update later
 		}
 
-		Vector<int> block_sizes;
+		uint32_t last_block_size = write_max % block_size;
+
+		// Temporary buffer for compressed data blocks.
+		Vector<uint8_t> temp_cblock;
```

Comment:
This should just be a LocalVector since it is only used temporarily. 

I don't expect that to have a measurable impact on performance, since you are basically just using the pointer to memory anyway. But its good code hygiene to use LocalVectors